sdk/core/azure-json-reflect/src/main/java/com/azure/json/reflect/JsonFactory.java

```diff
@@ -0,0 +1,17 @@
+package com.azure.json.reflect;
+
+import com.azure.json.JsonOptions;
+import com.azure.json.JsonWriter;
+import com.azure.json.JsonReader;
+
+import java.io.*;
+
+public interface JsonFactory {
```

Comment:
We are going to want a different name here I think. I don't know a good one off the top of my head, maybe @alzimmermsft has some thoughts.